initfiles/componentfiles/configxml/roxie.xsd.in

```diff
@@ -1211,6 +1211,27 @@
         </xs:appinfo>
       </xs:annotation>
     </xs:attribute>
+    <xs:attribute name="mysqlCacheCheckPeriod" type="xs:boolean" use="optional" default="10000">
```

Comment:
All 3 attributes are declared as xs:boolean, but they seem to be numeric

Reply:
Oops, too much cut/paste! I will fix